Source/WebCore/dom/Touch.cpp

```diff
@@ -29,68 +29,52 @@
 
 #include "Touch.h"
 
+#include "FloatPoint.h"
 #include "LocalDOMWindow.h"
 #include "LocalFrame.h"
 #include "LocalFrameView.h"
 
 namespace WebCore {
 
-static int contentsX(LocalFrame* frame)
+static FloatPoint contentsOffset(LocalFrame* frame)
 {
     if (!frame)
-        return 0;
+        return FloatPoint();
```

Comment:
Nit: We have a preference for brace initialization when possible.
```suggestion
        return { };
```